localstack/services/awslambda/lambda_executors.py

```diff
@@ -423,7 +438,14 @@ class LambdaExecutorContainers(LambdaExecutor):
     """Abstract executor class for executing Lambda functions in Docker containers"""
 
     def execute_in_container(
-        self, func_details, env_vars, command, docker_flags=None, stdin=None, background=False
+        self,
+        func_details,
+        handler,
```

Comment:
Wondering if we should refactor this method to passing in `InvocationContext` here. It's probably fine to merge this PR as-is, but we'd want to make this change fairly soon, to have better encapsulation (and extensibility) of this function..